src/app/shared/components/download/download.component.scss

```diff
@@ -56,44 +56,59 @@
         font-size: $font-size-x-regular;
     }
 
-    p-checkbox {
-        @include lato-font('normal');
-
-        line-height: normal;
-        font-size: $font-size-regular;
-        height: 32px;
-        color: color(text, secondary);
-
+    .ui-radiobutton-label {
+        font-size: 16px;
     }
 
-    .ui-chkbox {
-        margin-right: 17px;
-    }
+    .ui-radiobutton-box {
+        width: 15px;
+        height: 15px;
+        font-size: 12px;
+        border: 1px solid color(border);
+        border-radius: 50%;
+        transition: $transition-duration;
+
+        .ui-radiobutton-icon {
+            display: flex !important;
+            width: 100%;
+            height: 100%;
+            align-items: center;
+            justify-content: center;
+            transition: $transition-duration;
+            top: 0 !important;
```

Comment:
Heads up, using !important is widely discouraged.  It can be used in desperation, but in almost all cases we can devise a css selector whose specificity will cause it to take effect.

Reply:
:+1: